### PR TITLE
Closed loop flywheel control

### PR DESCRIPTION
**Pull request by outsidermm**

### CODE REVIEW

**Comment by outsidermm** [comment]
TODO: 
Change the max speed of the flywheel (in rot/s)
Tune the gains

Do we want velocity with voltage compensation, rn it’s velocity without voltage compensation

**Comment by outsidermm** [comment]
Feel free to work on this pr, I will not be here today :/

File: components/shooter.py [hunk 1]
```diff
@@ -12,10 +12,11 @@
 
 
 class ShooterComponent:
+    MAX_FLYWHEEL_SPEED = 6380 / 60
```
Comment (auscompgeek) [inline components/shooter.py:15]: This looks suspiciously like a motor free speed. Don't we have that defined in `utilities.ctre` already?

Comment (auscompgeek) [inline components/shooter.py:15]: Shouldn't this take into account the gearing?

File: robot.py [hunk 2]
```diff
@@ -113,7 +113,8 @@ def testPeriodic(self) -> None:
             self.intake.intake()
 
         if self.gamepad.getAButton():
-            self.shooter_component.set_inclination(math.radians(self.inclination_angle))
+            # self.shooter_component.set_inclination(math.radians(self.inclination_angle))
+            self.shooter_component.shoot()
```
Comment (auscompgeek) [inline robot.py:117]: Above, the `getBButton` if statement has a comment "injecting". I think that one might be intended to be this?

**Review by auscompgeek — APPROVED** [review]
[approveyoda](https://app.graphite.dev/macros/thedropbears)

**Pull request merged**